R/watcher.R

```diff
@@ -67,6 +80,22 @@ watchout <- function(handler = new_output_handler(),
     capture_output()
   }
 
+  print_value <- function(value, visible) {
```

Comment:
It feels a little weird to have this here, but the watcher is the one object that has all the details to handle this correctly.